src/Modules/Workspaces/Commands.Workspaces.Test/GetPowerBIWorkspaceTests.cs

```diff
@@ -426,6 +551,30 @@ public void GetWorkspacesOrganizationScopeSupportsName()
             AssertExpectedUnitTestResults(expectedWorkspaces, initFactory);
         }
 
+        [TestMethod]
+        public void GetWorkspacesOrganizationScopeSupportsAllWithName()
+        {
+            // Arrange
+            var name = "Test";
+            var filter = $"tolower(name) eq '{name.ToLower()}'";
+            var expectedWorkspaces = new List<Workspace> { new Workspace { Id = Guid.NewGuid(), Name = "TestWorkspace" } };
```

Comment:
Should Name match name variable? 
You should setup the mock to expect that variable name to be passed in so we verify it gets passed all the way through